mllib/src/test/scala/org/apache/spark/ml/feature/SQLTransformerSuite.scala

```diff
@@ -43,6 +43,7 @@ class SQLTransformerSuite
     assert(result.schema.toString == resultSchema.toString)
     assert(resultSchema == expected.schema)
     assert(result.collect().toSeq == expected.collect().toSeq)
+    assert(original.sparkSession.catalog.listTables().count() == 0)
```

Comment:
No, I believe `isEmpty` is only currently defined for an RDD, not a Dataset, you could do `collect().isEmpty` but that would be equally verbose I think.